scripts/knative-install.sh

```diff
@@ -28,3 +28,13 @@ data:
 EOF
 
 kubectl wait --for=condition=available --timeout=600s deployment/controller -n knative-serving
+
+# Patch the KNative domain configuration with the .omg.howdoi.website domain matching the Istio
+# ingress gateway load balancer IP address
+ISTIO_LB_IP=$(kubectl -n istio-system get service istio-ingressgateway -o=jsonpath='{.status.loadBalancer.ingress[0].ip}')
+if [ -z "$ISTIO_LB_IP" ]; then
+  echo "WARNING: could not update Knative domain configuration with a domain matching the Istio ingress gateway load balancer IP address !"
+else
+  kubectl -n knative-serving patch configmap config-domain --type merge \
+    -p "{\"data\":{\"$ISTIO_LB_IP.omg.howdoi.website\":\"\"}}"
```

Comment:
This won't work if I am using a different domain for FuseML. It might be better to get the domain from fuseml instead of hardcoding it to `.omg.howdoi.website`.

Reply:
I don't need the domain used by FuseML here, just the Istio one. Even if you're using a different domain for FuseML, the Istio installation doesn't care about that.

Reply:
Ok, agreed to extract the domain from FuseML, in cases like k3d where you don't get a load-balancer, or any other cases where the user installs Istio before fuseml and supplies a custom domain to `fuseml-installer`. I'll still use the Istio ingress gateway LB IP address as a backup, in case FuseML is still installed with Traefik instead of Istio.